.github/workflows/finishReleaseCycle.yml

```diff
@@ -91,7 +106,7 @@ jobs:
 
   # Update the staging branch to trigger a staging deploy
   updateStaging:
-    needs: [updateProduction, createNewPatchVersion]
+    needs: [cleanupDeployBlockers, createNewPatchVersion]
```

Comment:
Because `cleanupDeployBlockers` needs `updateProduction`, we'll still be waiting for `updateProduction` to finish before running `updateStaging`

Reply:
But wouldn't leaving what was before allow `updateStaging` to start at the same time as `cleanupDeployBlockers`? Anyway, NAB.

Reply:
But wouldn't leaving what was before allow updateStaging to start at the same time as cleanupDeployBlockers? Anyway, NAB.

Reply:
We should revert this line, clean up doesn't need to block staging deploy https://expensify.slack.com/archives/C07J32337/p1743687795684829